Rakefile

```diff
@@ -44,4 +36,22 @@ task :check_binstubs do
   end
 end
 
-task default: args
+Kitchen::RakeTasks.new
+
+desc 'Alias for kitchen:all'
```

Comment:
@majormoses What do you think of this for running kitchen tests concurrently?

Cut run time in half on my machine: `bundle exec rake integration  7.59s user 4.01s system 2% cpu 7:11.37 total`

Having a single Redis instance that all Kitchen suites use sounds like a good idea, however I'm not sure how to achieve that to work with travis too as I thought all the Travis runs were isolated from each other?

Reply:
> What do you think of this for running kitchen tests concurrently?
Makes sense for local for travis since they are a matrix job anyways there is no value of having them all run all suites when we can split it up into each job.

> Having a single Redis instance that all Kitchen suites use sounds like a good idea, however I'm not sure how to achieve that to work with travis too as I thought all the Travis runs were isolated from each other?
Yes they are isolated and really what I had in mind makes more sense when looking to having multiple of the same thing running in the container needing to bootstrap all those things takes time. It is probably faster to just spin up another docker image and link to it. This is especially helpful in a scenario when you are say wanting to test multiple versions of a backend. here is an example of doing it: https://github.com/sensu-plugins/sensu-plugins-postgres/pull/43 alternatively if there was some publicly accessible one available we could leverage existing infra. That being said I would never recommend exposing something like redis, mysq, etc, to the world. Something like nginx though would be fine.